src/lib/libast/include/ast_iconv.h

```diff
@@ -0,0 +1,85 @@
+/* : : generated from iconv by iffe version 2013-11-14 : : */
+#pragma prototyped
+#ifndef _def_iconv_features
+#define _def_iconv_features	1
+#define _sys_types	1	/* #include <sys/types.h> ok */
+#define _hdr_iconv	1	/* #include <iconv.h> ok */
+#define _lib_iconv_open	1	/* iconv_open() in default lib(s) */
+#define _lib_iconv_close	1	/* iconv_close() in default lib(s) */
+#define _lib_iconv	1	/* iconv() in default lib(s) */
+#define _nxt_iconv "../include/iconv.h"	/* include path for the native <iconv.h> */
+#define _nxt_iconv_str "../include/iconv.h"	/* include string for the native <iconv.h> */
+#include <ast_common.h>
+#include <ccode.h>
+#include <../include/iconv.h>	/* the native iconv.h */
```

Comment:
Now that this header has a distinct name that doesn't conflict with the system header this should just be `#include <iconv.h>` and the `_nxt_iconv` stuff above removed.